front/types/index.ts

```diff
@@ -70,6 +70,7 @@ export * from "./shared/utils/global_error_handler";
 export * from "./shared/utils/hashing";
 export * from "./shared/utils/iots_utils";
 export * from "./shared/utils/string_utils";
+export * from "./shared/utils/http_headers";
```

Comment:
Im' not sure this is a good place to add the util about HTTP headers, but I see other utils here